fix(statics): update zetachain to zetaevm

evalenti-simtlix opened the pull request:
update zetachain to zetaevm

EA-3126

TICKET: EA-3126

<!--
# Please be aware of the following when making your pull request:

## Description

Please include a summary of your proposed changes and which issue is being addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Issue Number

Internal Users - Please include the related internal tracking number (e.g. BG-000000).
External Users - Please link to any relevant github issues as necessary.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code compiles correctly for both Node and Browser environments
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My commits follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and I have properly described any BREAKING CHANGES
- [ ] The ticket or github issue was included in the commit message as a reference
- [ ] I have made corresponding changes to the documentation and on any new/updated functions and/or methods - [jsdoc](https://jsdoc.app/)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
-->

hitansh-bitgo commented:
Should be marked as breaking change in the commit footer

ajays97 commented on modules/statics/src/base.ts:
```diff
@@ -1521,7 +1521,7 @@ export enum UnderlyingAsset {
   ZCO = 'zco',
   ZECBEAR = 'zecbear',
   ZECBULL = 'zecbull',
-  ZETACHAIN = 'zetachain',
```
What are the consequences of removing this ENUM value downstream? Can we please check

evalenti-simtlix replied:
I think the consequences of removing is anyone, because i'm updating this one. But i need to update in Indexer and BGMS also.

alebusse reviewed (changes requested):
Mark it as breaking change, i think this is gonna leave every transfer or already saved doc with the old coin name invalid, i dont know if we ever did a name change like this

evalenti-simtlix committed:
fix(statics): update zetachain to zetaevm

update zetachain to zetaevm

EA-3126

TICKET: EA-3126

BREAKING CHANGE: we modify the symbol of the coins, and need to update this in indexer and wp also
to avoid errors and inconsistances

mmcshinsky-bitgo reviewed (changes requested):
Badly formatted commit. BREAKING CHANGE must be in the commit footer.
